torch/csrc/jit/codegen/cuda/fusion_segmenter.cpp

```diff
@@ -951,7 +951,7 @@ GroupDependencyAnalysis::GroupSet GroupDependencyAnalysis::getCommonProducersOf(
 
   // Get intersection of producers
   GroupSet common_producers = *(known_producers_of_.at(groups[0]));
-  for (size_t i = 1; i < groups.size(); i++) {
+  for (const auto i : c10::irange(groups.size())) {
```

Comment:
`c10::irange(1, groups.size())`?

Reply:
Yes it's intersection. Starting with 0 would give redundant compute.